### PR TITLE
add branding

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

File: home/templates/wagtailadmin/admin_base.html [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{% extends "wagtailadmin/admin_base.html" %}
+{% load wagtailsettings_tags static %}
```
Comment (rudigiesler) [inline home/templates/wagtailadmin/admin_base.html:2]: Do we need to load static?

File: home/templates/wagtailadmin/admin_base.html [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+
+
+  {% block branding_favicon %}
+      {% if settings.home.SiteSettings.favicon %}
+        <link rel="shortcut icon" href={{settings.home.SiteSettings.favicon.url }} />
+      {% else %}
+        {{ block.super}}
+      {% endif %}
+  {% endblock %}
+{% block branding_welcome %}{{ settings.home.SiteSettings.title }}{% endblock %}
```
Comment (rudigiesler) [inline home/templates/wagtailadmin/admin_base.html:13]: Should we have a default here if `title` isn't set in the settings?

File: home/templates/wagtailadmin/base.html [hunk 3]
```diff
@@ -1,10 +1,12 @@
...
-{% load wagtailadmin_tags wagtailcore_tags i18n static %}
-
+{% load wagtailadmin_tags wagtailcore_tags i18n static wagtailsettings_tags%}
+{% get_settings %}
 {% block furniture %}
     <link rel="stylesheet" type="text/css" href="{% static 'css/contentrepo.css' %}">
-    <template data-wagtail-sidebar-branding-logo>{% block branding_logo %}{% endblock %}</template>
+    <template data-wagtail-sidebar-branding-logo>{% block branding_logo %}
+    <img src={{settings.home.SiteSettings.logo.url }} alt={{settings.home.SiteSettings.title}} width="80" />
+{% endblock %}</template>
```
Comment (rudigiesler) [inline home/templates/wagtailadmin/base.html:9]: And a default for this image here, if it's not set?

**Review by KaitCrawford — APPROVED** [review]
👍

**Review by rudigiesler — APPROVED** [review]

**Pull request merged**